README.md

```diff
@@ -4,32 +4,36 @@
 
 No Installation required
 
-
-## TODO
-
-- Building instructions
-- Methodology
-
 ## Organizing CSS properties
 
 To keep every CSS declaration readable, let's follow some rules _(TODO: find a linter)_
 
 - put spaces at the proper spaces
```

Comment:
What does this mean?
Is it to say _2 spaces indent_?

Reply:
nope it's when you declare some properties:

.my-class {
 my-property: value;
}